Dockerfile

```diff
@@ -1,31 +1,38 @@
-FROM debian:stable-slim AS mirrorbits
+FROM golang:1.23.0 AS build
 
 ## (DL3008)Ignore lint error about apt pinned packages, as we always want the latest version of these tools
 ## and the risk of a breaking behavior is evaluated as low
 # hadolint ignore=DL3008
-RUN apt-get update && \
-  apt-get install --no-install-recommends -y tar curl ca-certificates && \
+RUN apt-get -qq update && \
+  apt-get install --no-install-recommends -y libgeoip-dev tar curl ca-certificates git unzip zlib1g-dev && \
   apt-get clean && \
   rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
-ARG mirrorbits_version=v0.5.1
-RUN mkdir /mirrorbits && \
-  curl -L https://github.com/etix/mirrorbits/releases/download/${mirrorbits_version}/mirrorbits-${mirrorbits_version}.tar.gz -O && \
-  tar xvzf /mirrorbits-${mirrorbits_version}.tar.gz -C /
+# 955a8b2e1aacea1cae06396a64afbb531ceb36d4 introduced go modules (for Go >= 1.11) and commited the generated code from protoc
+# v0.5.1 tag is a bit older (but without these 2 majors elements): https://github.com/etix/mirrorbits/commit/e83e56ac6496a3643d18a731324bd266f75f7b32 commit
+# Diff: https://github.com/etix/mirrorbits/compare/e83e56ac6496a3643d18a731324bd266f75f7b32..955a8b2e1aacea1cae06396a64afbb531ceb36d4
+ARG mirrorbits_version=955a8b2e1aacea1cae06396a64afbb531ceb36d4
 
-ARG tini_version=v0.19.0
-RUN curl --silent --show-error --output /mirrorbits/tini --location \
-  "https://github.com/krallin/tini/releases/download/${tini_version}/tini-$(dpkg --print-architecture)" && \
-  chmod +x /mirrorbits/tini
+WORKDIR "/mirrorbits"
 
-FROM debian:stable-slim
+# hadolint ignore=DL3003
+RUN git clone https://github.com/etix/mirrorbits ./ && \
```

Comment:
any reason why not to just use the version from debian unstable?
https://buildd.debian.org/status/package.php?p=mirrorbits

https://github.com/etix/mirrorbits/issues/150#issuecomment-1788362735

Reply:
Good point: did not know about https://github.com/etix/mirrorbits/issues/150#issuecomment-1788362735. That could be interesting to use their package.